pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/pendingack/impl/PendingAckHandleState.java

```diff
@@ -32,14 +32,15 @@
         None,
         Initializing,
         Ready,
+        Error,
```

Comment:
how can we recover from "Error" ?

Reply:
Different errors should be handled differently, but now I don’t know what error will happen, so just to be on the safe side, when error happen can't recover. unless you unload this topic